src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1677,6 +1681,26 @@ class DocBaseViewer extends BaseViewer {
         // in which the default discoverability experience is enabled
         this.containerEl.setAttribute(DISCOVERABILITY_ATTRIBUTE, isUsingDiscoverability);
     }
+
+    /**
+     * Hides the create region cursor popup for a document
+     *
+     * @protected
+     * @return {void}
+     */
+    applyCursorFtux() {
+        if (!this.containerEl) {
+            return;
+        }
+
+        if (this.annotationControlsFSM.getState() === AnnotationState.REGION) {
```

Comment:
Could we move this into the guard clause above? then we can flatten out this nested if-else statement